src/main/java/uk/gov/hmcts/ccd/data/definition/CachedCaseDefinitionRepository.java

```diff
@@ -49,6 +47,18 @@ public UserRole getUserRoleClassifications(String userRole) {
         return userRoleClassifications.computeIfAbsent(userRole, caseDefinitionRepository::getUserRoleClassifications);
     }
 
+    @Override
+    public List<UserRole> getClassificationsForUserRoleList(List<String> userRoles) {
+        final List<String> missingRoles = userRoles.stream()
```

Comment:
can we rename missingRoles to missingClassifications?

Reply:
Actually that name is used a few lines below that one